spec/controllers/api/v3/users_controller_spec.rb

```diff
@@ -155,6 +155,19 @@
       user.reload
       expect(user.otp).not_to eq(existing_otp)
     end
+
+    it "uses a sensible default when the OTP delay is not configured" do
+      ENV["USER_OTP_SMS_DELAY_IN_SECONDS"] = nil
+
+      existing_otp = user.otp
+
+      allow(RequestOtpSmsJob).to receive_message_chain("set.perform_later")
+      expect(RequestOtpSmsJob).to receive_message_chain("set.perform_later").with(user)
```

Comment:
Why do we need the `allow` if there's already an `expect`?

Reply:
Yup you're right. We don't really need em. Axed.